spec/definitions/prepair_host.sh

```diff
@@ -0,0 +1,11 @@
+#update git
+cd ~/machinery/ && git pull
+
+#build package
+cd ~/machinery/ && rake rpm:build[https://api.suse.de,Devel:Alfred,SLE_12]
+
+#install package
+package=$(find ~/machinery/package/ \
+  -name "machinery-[0-9]*.ppc64le.rpm" -or \
+  -name "machinery-[0-9]*.s390x.rpm")
```

Comment:
The Problem is there is also a package named machinery-*.src.rpm and a wildcard will include them. We can filter them but I think this version is more robust.